Makefile

```diff
@@ -146,7 +137,15 @@ run_bash:
 		-p 8888:8888 \
 		$(DOCKER_STOCHSS_IMAGE):latest \
 		/bin/bash
-
+watch:
+	docker run -it --rm \
+		--name $(DOCKER_STOCHSS_IMAGE) \
+		--env-file .env \
+		-v $(PWD):/stochss \
```

Comment:
`$(PWD)` seems to evaluate to an empty string when running the Makefile on Windows. Both `run` and `watch` fail because of this.

The .env file evaluates `$(PWD)/local_data` to `/local_data`, and the `docker` volume mount evaluates `-v $(PWD):/stochss` to `-v :/stochss`.